src/OpenTelemetry/Metrics/MetricType.cs

```diff
@@ -19,16 +19,29 @@
 namespace OpenTelemetry.Metrics
 {
     [Flags]
-    public enum MetricType : byte
+    public enum MetricType : int
     {
         /*
-        Type:
-            0x10: Sum
-            0x20: Gauge
-            0x30: Histogram
-            0x40: Summary (reserved)
+        Type (bits 4,5,6,7,8,9,10,11):
```

Comment:
We need to have some consistency here, the spacing, whether binary/hex are used, etc.